cmd/commands/run.go

```diff
@@ -29,6 +29,7 @@ func NewCmdRun() *cobra.Command {
 		Run:     doRun,
 	}
 
+	cmd.Flags().String("event", "", "Specifies the event trigger to use if there are multiple configured")
```

Comment:
Could use `StringP` for a `-e` arg shorthand. I used `event` instead of `trigger` b/c we use `event` for `inngest init`